etchosts/etchosts_test.go

```diff
@@ -137,6 +137,103 @@ func TestUpdate(t *testing.T) {
 	}
 }
 
+// Regression test for GitHub issue #603 on Update
```

Comment:
Please add a brief note about the test rather than referring to the issue number here.

Can you also squash the two commits into one.